meveo-admin-ejbs/src/main/java/org/meveo/service/custom/CustomEntityTemplateService.java

```diff
@@ -920,4 +920,12 @@ public Logger getLogger() {
 		return log;
 	}
 
+    public String getModuleBelonging(String cetCode) {
```

Comment:
@mtnhan there is already an existing method : BusinessService#findModuleOf

Reply:
Refactor code for using findModuleOf and fixed issue of the filter.